src/modules/lost/doc/lost_admin.xml

```diff
@@ -290,6 +399,57 @@ xlog("L_INFO", "HELD locationRequest: Result code $var(res)\nUrl: $var(url)\n$va
 				</programlisting>
 			</example>
 		</section>
+		<section id="lost.f.lost_held_dereference">
+			<title>
+				<function moreinfo="none">lost_held_dereference(url, rtime, rtype, pidf-lo, error)</function>
+			</title>
+			<para>
+			Sends a HELD POST locationRequest to a given URL. Attributes are responseTime and resposeType.
+	    	</para>
+			    <itemizedlist>
+                    <listitem><para>
+						<emphasis>url</emphasis> - a URL received via Geolocation header to dereference
+                        location
+                    </para></listitem>
+				    <listitem><para>
+						<emphasis>rtime</emphasis> - the response time as defined
+                        in <xref linkend="lost.p.response_time"/>
+				    </para></listitem>
+                    <listitem><para>
+						<emphasis>rtype</emphasis> - the response type (location) as defined
+                        in <xref linkend="lost.p.location_type"/>
+                    </para></listitem>
+                    <listitem><para>
+						<emphasis>pidf-lo</emphasis> - the PIDF-LO returned in the
+                        HELD locationRequest response
+                    </para></listitem>
+                    <listitem><para>
+						<emphasis>error</emphasis> - any error code returned in the
+                        HELD response
+                    </para></listitem>
+			    </itemizedlist>
+			<para>
+			The return value is 200 on success, 400 if an internal error occured, or 500 if an
+            error code is returned in the HELD response.
+	    	</para>
+			<para>
+			This function can be used from REQUEST_ROUTE,
+			ONREPLY_ROUTE, FAILURE_ROUTE, and BRANCH_ROUTE.
+			</para>
+			<example>
+				<title><function>lost_held_dereference()</function> usage</title>
+				<programlisting format="linespecific">
+...
+# HELD location dereference
+if ($hdr(Geolocation)=~"^&lt;http.*$") {
+    $var(url) = $(hdr(Geolocation){s.select,0,;});
```

Comment:
I can suggest use
```
$var(url) = @hf_value.geolocation[1].uri;
```

Reply:
Note that `@...` were SER specific variables, still available, but not with an active maintainer. I would rather recommend using `$...` instead of `$...`, so better examples with $hdr(...) when gives what's needed.

Reply:
@sergey-safarov: you may use the following to dereference multiple header fields:
```
## response time in ms: 20000, or 'emergencyRouting' or 'emergencyDispatch' 
#!substdef "!MY_LOC_TIME!emergencyDispatch!g"
## location type: 'any', 'civic', 'geodetic', or 'civic geodetic'
#!substdef "!MY_LOC_TYPE!any!g"
```
...
```
if(is_present_hf("Geolocation")) {
        $var(count) = 0;
        while($var(count) < $hdrc(Geolocation)) {
                $var(geo) = $(hdr(Geolocation)[$var(count)]);
                if ($var(geo)=~"^<http.*$") {
                        $var(url) = $(var(geo){s.select,0,;});
                        xlog("L_INFO", "### HELD dereferencing location at: $(var(url){s.unbracket})\n");
                        $var(res) = lost_held_dereference("$(var(url){s.unbracket})", "MY_LOC_TIME", "MY_LOC_TYPE", "$var(pidf)", "$var(err)");
                        if ($var(res) == 200) {
                                xlog("L_INFO", "### HELD dereference request returned loc:\n$var(pidf)\n");
                        } else {
                                xlog("L_INFO", "###  held request failed with $var(err)\n");
                        }
                }
        }
}
```